homeassistant/components/tts/__init__.py

```diff
@@ -47,6 +47,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+TtsAudioType = Tuple[str, bytes]
```

Comment:
I think this is closer to what `tts` expects.
```suggestion
TtsAudioType = Tuple[Optional[str], Optional[bytes]]
```